less/styles.less

```diff
@@ -211,6 +211,26 @@ html.no-js .sidebar {
     padding: @grid-gutter-width / 2;
   }
 
+  .sidebar-menu li img{
+    float: none;
+    width: 5.4rem;
+    opacity: 0.6;
+    margin: 0 1.5rem 0 0;
+    @media (min-width: @screen-md-min) {
+      float: left;
+    }
+  }
+
+  .sidebar-menu li:nth-child(3) img {
```

Comment:
Hmm, I am a bit wary of `:nth-child` because changing the DOM structure just a little bit could throw off the styling in hard-to-debug ways. Do you think maybe we could add a `.sidebar-icon` class instead? Unless `:nth-child` is an accepted best practice and I'm just being paranoid, which is entirely possible. I guess I'd defer to @gesa for best judgement on this.